### PR TITLE
Translate security/_index into chinese

### PR DESCRIPTION
**Pull request by xieydd**
Translated content/docs/concepts/security/_index.mdinto Chinese.

### CODE REVIEW

File: content_zh/docs/concepts/security/_index.md [hunk 1]
```diff
@@ -4,3 +4,216 @@ description: 描述 Istio 的授权与认证功能。
...
+    caption="Istio RBAC Architecture"
+    >}}
+
+Istio的RBAC引擎做了下面两件事：
+* **获取RBAC策略.** Istio RBAC引擎关注RBAC策略的变化。如果它看到任何更改，它将获取更新后的RBAC策略。
+* **授权请求.** 在运行时，当一个请求到来时，请求上下文被传递给Istio RBAC引擎。RBAC引擎根据内容对环境进行评估，并返回授权结果（允许或拒绝）。
+
+### 请求上下文
+
+在当前版本中，Istio RBAC引擎被实现为一个[混合适配器](https://github.com/istio/istio.github.io/blob/master/docs/concepts/policies-and-telemetry/#adapters)。请求上下文是作为[授权模板](https://github.com/istio/istio.github.io/blob/master/docs/reference/config/policy-and-telemetry/templates/authorization)的一个实例提供的。请求上下文包含关于请求和授权模块需要知道的环境的所有信息。特别是，它有两个部分：
```
Comment (gyliu513) [inline content_zh/docs/concepts/security/_index.md]: 需要链接中文内容

File: content_zh/docs/concepts/security/_index.md [hunk 2]
```diff
@@ -4,3 +4,216 @@ description: 描述 Istio 的授权与认证功能。
...
+* 基于角色的语义，它简单易用。
+* 服务到服务以及用户端到服务的授权。
+* 在角色和角色绑定中可以通过自定义属性保证灵活性。
+
+## 架构
+
+下面的图表显示了Istio RBAC体系结构。操作者指定Istio RBAC策略。策略保存在Istio配置存储中。
+
+{{< image width="80%" ratio="56.25%"
+    link="./IstioRBAC.svg"
```
Comment (gyliu513) [inline content_zh/docs/concepts/security/_index.md]: image路径需要更新下

**Comment by xieydd** [comment]
i sign it!

**Comment by xieydd** [comment]
@fleeto I will check carefully later.

File: content_zh/docs/concepts/security/_index.md [hunk 3]
```diff
@@ -4,3 +4,217 @@ description: 描述 Istio 的授权与认证功能。
 weight: 30
 type: section-index
 ---
+
+## 综述
+
+Istio 基于角色的访问控制（RBAC）为 Istio 网格中的服务提供名称空间级、服务级、方法级访问控制。它的特点:
```
Comment (rootsongjc) [inline content_zh/docs/concepts/security/_index.md]: 统一词汇翻译，namespace
名称空间 -> 命名空间

File: content_zh/docs/concepts/security/_index.md [hunk 4]
```diff
@@ -4,3 +4,217 @@ description: 描述 Istio 的授权与认证功能。
...
+* 基于角色的语义，它简单易用。
+* 服务到服务以及用户端到服务的授权。
+* 在角色和角色绑定中可以通过自定义属性保证灵活性。
+
+## 架构
+
+下面的图表显示了 Istio RBAC 体系结构。操作者可以指定 Istio RBAC 策略。策略则保存在 Istio 配置存储中。
+
+{{< image width="80%" ratio="56.25%"
+    link="https://github.com/istio/istio.github.io/tree/master/content/docs/concepts/security/IstioRBAC.svg"
```
Comment (fleeto) [inline content_zh/docs/concepts/security/_index.md]: link="https://github.com/istio/istio.github.io/tree/master/content/docs/concepts/security/IstioRBAC.svg"
=>
link="/docs/concepts/security/IstioRBAC.svg"

File: content_zh/docs/concepts/security/_index.md [hunk 5]
```diff
@@ -4,3 +4,217 @@ description: 描述 Istio 的授权与认证功能。
...
+    caption="Istio RBAC Architecture"
+    >}}
+
+Istio 的 RBAC 引擎做了下面两件事：
+* **获取 RBAC 策略.** Istio RBAC 引擎关注 RBAC 策略的变化。如果它看到任何更改，将会获取更新后的 RBAC 策略。
+* **授权请求.** 在运行时，当一个请求到来时，请求上下文会被传递给 Istio RBAC 引擎。RBAC 引擎根据传递的内容对环境进行评估，并返回授权结果（允许或拒绝）。
+
+### 请求上下文
+
+在当前版本中，Istio RBAC 引擎被实现为一个[Mixer 适配器](https://github.com/istio/istio.github.io/tree/master/content_zh/docs/concepts/policies-and-telemetry#adapters)。请求上下文则作为[授权模板](https://github.com/istio/istio.github.io/tree/master/content/docs/reference/config/policy-and-telemetry/templates/authorization/)的实例。请求上下文包含请求和授权模块需要环境的所有信息。特别是两个部分：
```
Comment (fleeto) [inline content_zh/docs/concepts/security/_index.md]: https://github.com/istio/istio.github.io/tree/master/content_zh/docs/concepts/policies-and-telemetry#adapters
=>
/docs/concepts/policies-and-telemetry#adapters

**Comment by fleeto** [comment]
先抛开字句和链接问题。这个似乎是用 RBAC 的内容覆盖了整个 Security ？

File: content_zh/docs/concepts/security/_index.md (hunk 5 above)
Comment (xieydd) [reply content_zh/docs/concepts/security/_index.md]: I have try relative path,but i can`t pass CI

**Comment by xieydd** [comment]
@fleeto I find `Consolidate the security concept pages into a single page.` #1721  .what should i do, keep it sync to this commit.

**Pull request closed without merging**